mlrun/feature_store/api.py

```diff
@@ -405,7 +408,12 @@ def ingest(
         raise mlrun.errors.MLRunInvalidArgumentError(
             "feature set and source must be specified"
         )
-
+    entities_keys = [e.name for e in featureset.spec.entities]
+    for step_name, step_value in featureset.graph.steps.items():
+        if step_name == get_name(name=None, class_name=DropFeatures):
+            dropped_features = step_value.class_args.get("features")
+            if set(dropped_features).intersection(set(entities_keys)):
```

Comment:
No need to convert to set.
```suggestion
            if set(dropped_features).intersection(entities_keys):
```

Reply:
fixed! ty!